src/gui/MainPane/RoomDelegate.qml

```diff
@@ -63,7 +63,9 @@ HTile {
                 spacing: room.spacing
 
                 TitleLabel {
-                    text: model.display_name || qsTr("Empty room")
+                    text:
+                        (model.bookmarked ? "\u2764 " : "") +
```

Comment:
```suggestion
                        (model.bookmarked ? "\u2665 " : "") +
```
[2764](https://unicode-table.com/en/2764/) looks vertically squished with the default app font (Roboto), [2665](https://unicode-table.com/en/2665/) feels more natural.